docs/config.rst

```diff
@@ -0,0 +1,778 @@
+        steps:
+            1:
+                task: upload_beta
+                options:
+                    name: Automated beta release
+            2:
+                task: github_release
+                options:
+                    version: ^^upload_beta.version_number
+            3:
+                task: github_release_notes
+                ignore_failure: True  # Attempt to generate release notes but don't fail build
+                options:
+                    link_pr: True
+                    publish: True
+                    tag: ^^github_release.tag_name
+                    include_empty: True
+                    version_id: ^^upload_beta.version_id
+            4:
+                task: github_master_to_feature
+
+This flow shows how subsequent tasks can reference the return values of a prior task. In this case, the ``github_release`` task uses the ``version_numer`` set by the ``upload_beta`` task as an option value with the ``^^upload_beta.version_number`` syntax. Similarly, the ``github_release_notes`` task uses the ``version_id`` set by the ``upload_beta`` task as an option value with the ``^^upload_beta.version_id`` syntax.
+
+
+
+Tasks and Flows from a Different Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It's also possible to use tasks and flows from another project with CumulusCI. The other project must be named under the ``sources`` section of the project ``cumulusci.yml`` file.
+
+For example, when tasks or flows are referenced using the `npsp` namespace, CumulusCI fetches the source from the NPSP GitHub repository.
+
+.. code-block:: yaml
+
+    sources:
+      npsp:
+        github: https://github.com/SalesforceFoundation/NPSP
+
+By default, CumulusCI fetches the most recent release, or the default branch if there are no releases.
+
+.. note::
+
+    This feature requires that the referenced repository be readable (for example,
+    it's public, or CumulusCI's GitHub service is configured with the token of a user who has read access to it).
+
+
+It's also possible to fetch a specific ``tag``...
+
+.. code-block:: yaml
+
+    sources:
+      npsp:
+        github: https://github.com/SalesforceFoundation/NPSP
+        tag: rel/3.163
+
+or a specific ``commit`` or ``branch``.
+
+When the repo is listed under ``sources``, it's possible to run a task from NPSP...
+
+.. code-block:: console
+
+    $ cci task run npsp:robot
+
+Or a flow...
+
+.. code-block:: console
+
+    $ cci flow run npsp:install_prod
+
+Or even create a new flow that uses a flow from NPSP:
+
+.. code-block:: yaml
+
+    flows:
+        install_npsp:
+            steps:
+            1:
+                flow: npsp:install_prod
+            2:
+                flow: dev_org
+
+This flow uses NPSP's ``install_prod`` flow to install NPSP as a managed package, and then run this project's own ``dev_org`` flow.
+
+
+
+Implementation of Metadata ETL Tasks
```

Comment:
This section does not go here. I suggest either delete it entirely, or move it under the top-level Metadata ETL heading.

Reply:
I think the content is worth preserving. I'll put it in Metadata ETL.